publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -309,6 +309,76 @@ You can also see pods for the pipeline runs, for which you can specify `oc descr
 
 If the pipeline run was successful, you can see a Docker image in our Docker registry and a pod that’s running your application.
 
+## Transport layer security (TLS) verification for image registry access in pipelines
+
+when accessing image registries to pull and push images, pipelines enable the configuration creation of the cluster resource, [image.config.openshift.io/cluster](https://docs.openshift.com/container-platform/4.2/openshift_images/image-configuration.html#images-configuration-file_image-configuration).  Scenarios are presented of image registries that can be used and the setup steps required.
+
+### Enable TLS verification for image registry access:
+
+#### Private image registries:
+ 
+If you use a private image registry and your registry uses certificates that are signed by trusted CA authorities, no further configuration is needed to enable TLS verification. Review the default truststore on the nodes of your cluster to ensure that you have the CA of your certificate in the list. With a self-signed certificates or a certificate that is signed by a CA, you must ensure that either certificate is listed in the default truststore. Steps are presented to add the CA to the trusted list so that when you run the pipelines, these CA certificates in the list are used.
```

Comment:
I think we should update this line `With a self-signed certificates or a certificate that is signed by a CA, you must ensure that either certificate is listed in the default truststore.` to just be `With a self-signed certificates, you must ensure that the CA certificate is add to the appropriate config map as detailed below`.

Reply:
Updated as requested